ranger-bom/pom.xml

```diff
@@ -0,0 +1,175 @@
+      <dependency>
+        <groupId>io.appform.ranger</groupId>
+        <artifactId>ranger-drove</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.appform.ranger</groupId>
+        <artifactId>ranger-drove-client</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.appform.ranger</groupId>
+        <artifactId>ranger-http</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.appform.ranger</groupId>
+        <artifactId>ranger-http-client</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.appform.ranger</groupId>
+        <artifactId>ranger-http-model</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.appform.ranger</groupId>
+        <artifactId>ranger-http-server</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.appform.ranger</groupId>
+        <artifactId>ranger-http-server-bundle</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.appform.ranger</groupId>
+        <artifactId>ranger-hub-server-bundle</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.appform.ranger</groupId>
+        <artifactId>ranger-server</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.appform.ranger</groupId>
+        <artifactId>ranger-server-bundle</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.appform.ranger</groupId>
+        <artifactId>ranger-server-common</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.appform.ranger</groupId>
+        <artifactId>ranger-zk-client</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.appform.ranger</groupId>
+        <artifactId>ranger-zk-server</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.appform.ranger</groupId>
+        <artifactId>ranger-zk-server-bundle</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.appform.ranger</groupId>
+        <artifactId>ranger-zookeeper</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.jacoco</groupId>
+        <artifactId>jacoco-maven-plugin</artifactId>
```

Comment:
For this to work properly, i think, all modules need to be added as dependencies as well. Please check the generated jacoco report to ensure we are getting data for all modules

Reply:
Added dependencies in ranger-bom to generate aggregate jacoco report